### DISCUSSION TITLE
If we don't specify any query when using the get_all_tweets() function, what will we retrieve?

### DISCUSSION THREAD

**Discussion by Jacobzwj** [cjbarrie/academictwitteR#375] [Q&A]
For example, if we use the folllowing codes (no query):
tweets <-
  get_all_tweets(
    start_tweets = "2020-01-01T00:00:00Z",
    end_tweets = "2020-01-05T00:00:00Z" )

1. Will we receive all tweets within the designated time period, or 
2. Will the function only provide us with a random sample of tweets within that time frame? 

Thank you for your assistance!

Best,
Jacob

**Comment by wagenkle** [comment]
I guess option 1, you could test by first using the count_all_tweets function to obtain the number of tweets and then run the code as above. You could also make use of the n argument to not exceed your Twitter API cap.

**Reply by Jacobzwj** [reply]
Thanks for your advice. After testing, I think it should be the Option 1.
I got 4 datasets.

- dataset 1→period:  3 days; no query.
- dataset 2→period:  3 days; query: "am".
- dataset 3→period:  3 days; query: "is".
- dataset 4→period:  3 days; query: "are"

Finding:
dataset 2, 3, 4 are all the subsets of dataset 1.
Thus, Option 1 is probably correct.
